aspnetcore/migration/identity.md

```diff
@@ -63,7 +63,30 @@ namespace NewMvc6Project.Models
 
 ApplicationDbContext.cs:
 
-<!-- literal_block {"ids": [], "names": [], "highlight_args": {}, "backrefs": [], "dupnames": [], "linenos": false, "classes": [], "xml:space": "preserve", "language": "c#"} -->
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+```csharp
+using Microsoft.AspNetCore.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+
+namespace NewMvc6Project.Models
+{
+  public class ApplicationDbContext : IdentityDbContext<ApplicationUser>
+  {
+    public ApplicationDbContext()
+    {
+      Database.EnsureCreated();
```

Comment:
It is not good practice to try to create the database in the context constructor--it can have nasty side effects in tooling.